tests/labhub_test.py

```diff
@@ -280,12 +321,6 @@ def test_assign_cmd(self):
                               'not eligible to be assigned to this issue')
         testbot.pop_message()
 
-        # no assignee, newcomer, difficulty medium
-        labhub.GH_ORG_NAME = 'not-coala'
```

Comment:
https://github.com/coala/corobo/blob/de4fbaff1bd227d078f15e8221ea4941fdcf4450/plugins/labhub.py#L74 
it was resulting to a KeyError when `team_mapping`  was being called as labhub.GH_ORG_NAME was 'not-coala' instead of `coala`